### PR TITLE
Resolve front end bug

### PR DESCRIPTION
**Pull request by michaelecolley**
### Summary

Resolve frontend bug, implement search functionality and style website

### Changes being made

- Reformat `login.js`
- Remove in-line formatting
- Rename `MovieContainer` to `MovieSearchContainer`
- Introduce `regeneratorRuntime` to allow babel to work as expected
- Remove redundant img tag in `Main.jsx`
- Reinstate `index.js` to how it was before
- Update the css style sheet
- Removed script tag from `index.html`
- Commented out get request in `movieRouter.js`

### Validation tests

Visual check of the website, search functionality is operating as expected

### CODE REVIEW

File: client/containers/MovieSearchContainer.jsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect, useRef } from "react";
+import MovieTile from "../components/MovieTile";
+import axios from "axios";
+
+function MovieContainer(props) {
```
Comment (sar516) [inline client/containers/MovieSearchContainer.jsx]: change the name of this component to match name of file

File: client/index.js [hunk 2]
```diff
@@ -1,11 +1,10 @@
 import React from "react";
-import { render } from "react-dom";
+import ReactDOM from "react-dom";
 import App from "./App.jsx";
-import { BrowserRouter as Router } from "react-router-dom";
 
-render(
-  <Router>
```
Comment (sar516) [inline client/index.js:7]: Are we getting rid of router?...I'm not complaining

**Review by sar516 — COMMENTED** [review]
check out my comments in MovieSearchContainer.jsx and index.js

File: client/index.js (hunk 2 above)
Comment (michaelecolley) [reply client/index.js:7]: We have router in App.jsx

Comment (sar516) [reply client/index.js:7]: cool

File: client/containers/MovieSearchContainer.jsx (hunk 1 above)
Comment (michaelecolley) [reply client/containers/MovieSearchContainer.jsx]: fair, I have updated it.

**Review by sar516 — APPROVED** [review]
All good

**Pull request merged**